www/assets/js/components/SearchResult.tsx

```diff
@@ -118,6 +118,10 @@ function isResource(object: LearningResource): boolean {
 export function LearningResourceDisplay(props: SRProps) {
   const { object, id } = props
   const maxTags = 3
+  const runSlug = object.run_slug
+  const url = runSlug
+    ? `${runSlug.includes("courses/") ? "/" : "/courses"}${runSlug}`
```

Comment:
`runSlug` isn't going to include a preceding slash either way, so I think this should be:

```javascript
${runSlug.includes("courses/") ? "/" : "/courses/"}
```

Reply:
Thanks, this should be all set now.